examples/plot_clip.py

```diff
@@ -0,0 +1,135 @@
+"""
+Clip Vector Data with GeoPandas
+==================================================================
+
+Learn how to clip point, line, or polygon geometries to the boundary
+of a polygon geometry using GeoPandas.
+
+"""
+
+###############################################################################
+# Clip Vector Data in Python Using GeoPandas
+# ---------------------------------------------
+#
+# The example below shows you how to clip a set of vector geometries
+# to the spatial extent / shape of another vector object. Both sets of geometries
+# must be opened with GeoPandas as GeoDataFrames and be in the same Coordinate
+# Reference System (CRS) for the ``clip()`` function in EarthPy to work.
+#
+# This example uses Polygons, a Line, and Points made with shapely and then
+# turned into GeoDataframes.
+#
+# .. note::
+#    The object to be clipped will be clipped to the full extent of the clip
+#    object. If there are multiple polygons in clip object, the input data will
+#    be clipped to the total boundary of all polygons in clip object.
+
+###############################################################################
+# Import Packages
+# ---------------
+#
+# To begin, import the needed packages.
+
+import numpy as np
+import matplotlib.pyplot as plt
+import geopandas as gpd
```

Comment:
```suggestion
import geopandas
```

(we are trying to be consistent about it in our own docs to not abbreviate it ;))